content/blog/posts/osl-alumni-matthew-tesla.rst

```diff
@@ -0,0 +1,19 @@
+OSL Alumni Matthew Johnson on working at Tesla
+==============================================
+:date: 2019-03-07
+:slug: osl-alumni-matthew-johnson
+:author: Jerry Peng
+:img: matthew.jpg
+:tag: featured-stories
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/uHvRPlV0Fg8" frameborder="0"
+    allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Can you move this under the paragraph? I think it might look better there.

Reply:
Yea, sorry for pushing so many times. I can't start pelican for some reason.